### PR TITLE
Flp 61 earning calculation completion amount

### PR DESCRIPTION
**Pull request by ShakilUmar**

### CODE REVIEW

File: src/SFA.DAS.Funding.SystemAcceptanceTests/Configurator.cs [hunk 1]
```diff
@@ -20,7 +20,8 @@ private static IConfigurationRoot GetIConfigurationRoot()
             return new ConfigurationBuilder()
                 .AddEnvironmentVariables()
                 .SetBasePath(Directory.GetCurrentDirectory())
-                .AddJsonFile("local.settings.json", optional: true)
+                .AddJsonFile("appsettings.ProjectConfig.json", optional: true)
+                .AddUserSecrets("FundingTestExecutionSecrets")
```
Comment (ruslanss) [inline src/SFA.DAS.Funding.SystemAcceptanceTests/Configurator.cs:24]: is this needed?

File: src/SFA.DAS.Funding.SystemAcceptanceTests/Helpers/ApprenticeshipEntityApiClient.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+        public ApprenticeshipEntityApiClient(ScenarioContext context)
+        {
+            _context = context;
+            _config = _context.Get<FundingConfig>();
+            _earnings = _context.Get<EarningsGeneratedEvent>();
+        }
+
+        protected override string ApiBaseUrl => $"https://das-{_config.EnvironmentName}-funappern-fa.azurewebsites.net";
+
```
Comment (ruslanss) [inline src/SFA.DAS.Funding.SystemAcceptanceTests/Helpers/ApprenticeshipEntityApiClient.cs:19]: this won't work for localhost :/

File: src/SFA.DAS.Funding.SystemAcceptanceTests/SFA.DAS.Funding.SystemAcceptanceTests.csproj [hunk 3]
```diff
@@ -13,6 +13,7 @@
     <PackageReference Include="Microsoft.Extensions.Configuration.EnvironmentVariables" Version="6.0.1" />
     <PackageReference Include="Microsoft.Extensions.Configuration.FileExtensions" Version="6.0.0" />
     <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="6.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.UserSecrets" Version="6.0.1" />
```
Comment (ruslanss) [inline src/SFA.DAS.Funding.SystemAcceptanceTests/SFA.DAS.Funding.SystemAcceptanceTests.csproj:16]: is this needed?

File: src/SFA.DAS.Funding.SystemAcceptanceTests/Configurator.cs (hunk 1 above)
Comment (satya-sanikommu) [reply src/SFA.DAS.Funding.SystemAcceptanceTests/Configurator.cs:24]: @ruslanss Yes, we are going to read the config from the secrets moving forward for local execution, as we are now introducing some sensitive data (like connection keys)

File: src/SFA.DAS.Funding.SystemAcceptanceTests/SFA.DAS.Funding.SystemAcceptanceTests.csproj (hunk 3 above)
Comment (satya-sanikommu) [reply src/SFA.DAS.Funding.SystemAcceptanceTests/SFA.DAS.Funding.SystemAcceptanceTests.csproj:16]: Yes for the reason mentioned above

File: src/SFA.DAS.Funding.SystemAcceptanceTests/Helpers/ApprenticeshipEntityApiClient.cs (hunk 2 above)
Comment (satya-sanikommu) [reply src/SFA.DAS.Funding.SystemAcceptanceTests/Helpers/ApprenticeshipEntityApiClient.cs:19]: Fixed, pushed the updated code

**Review by satya-sanikommu — APPROVED** [review]

**Review by ruslanss — APPROVED** [review]

**Pull request merged**